medusa/process_tv.py

```diff
@@ -334,11 +334,9 @@ def validateDir(self, path, dirName, nzbNameOriginal, failed):
         :param result: Previous results
         :return: True if dir is valid for processing, False if not
         """
-        dirName = ss(dirName)
-
-        IGNORED_FOLDERS = [u'.AppleDouble', u'.@__thumb', u'@eaDir']
+        ignored_folders = [u'.AppleDouble', u'.@__thumb', u'@eaDir']
```

Comment:
Technically in python there aren't any constants.  It's more about intent, if `ignored_folders` is never intended to change values then it's "constant" and should be capitalized.  And you're right as the method is a class method it should be a class constant and not a module content.